packages/core/src/di/factoryMethods.ts

```diff
@@ -36,3 +36,16 @@ optionsFactory.inject = tokens<[typeof coreTokens.configReadFromConfigFile, type
   coreTokens.configReadFromConfigFile,
   coreTokens.configEditorApplier
 );
+
+export function mutatorDescriptorFactory(options: StrykerOptions): MutatorDescriptor {
```

Comment:
I would suggest to change the type of `mutator` on the `StrykerOptions` to be of type `mutator: string | Partial<MutatorDescriptor>`.

After that change, you'll get a compile error here, stating that `Type 'Partial<MutatorDescriptor>' is not assignable to type 'MutatorDescriptor'`. You can fix that by providing default values:
```ts
return {
    plugins: [],
    excludedMutations: [],
    name: '',
    ...options.mutator
  };
```